src/Uno.UI/UI/Xaml/Controls/ScrollViewer/ScrollViewer.macOS.cs

```diff
@@ -1,17 +1,164 @@
-﻿using Windows.UI.Xaml.Controls;
+﻿using Uno.Extensions;
+using Uno.UI.DataBinding;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Data;
 using System;
 using System.Collections.Generic;
-using System.Linq;
+using System.Drawing;
+using Uno.Disposables;
+using System.Runtime.CompilerServices;
 using System.Text;
-using System.Threading.Tasks;
-using Windows.UI.Xaml.Data;
-using Windows.UI.Xaml.Media;
-using Windows.UI.Xaml.Media.Animation;
+using System.Linq;
+
+using Foundation;
+using CoreGraphics;
+using Uno.UI.Extensions;
 
 namespace Windows.UI.Xaml.Controls
 {
-	public partial class ScrollViewer 
+	public partial class ScrollViewer : ContentControl
 	{
-		private void UpdateZoomedContentAlignment() { }
+		/// <summary>
+		/// On iOS 10-, we set a flag on the view controller such that the CommandBar doesn't automatically affect ScrollViewer content 
+		/// placement. On iOS 11+, we set this behavior on the UIScrollView itself.
+		/// </summary>
+		internal static bool UseContentInsetAdjustmentBehavior => false;
```

Comment:
The comment doesn't apply, and probably the property doesn't apply either?

Reply:
Indeed, removed.